src/main/java/org/softc/armoryexpansion/common/integration/aelib/plugins/constructsarmory/material/ArmorToolMaterial.java

```diff
@@ -0,0 +1,80 @@
+package org.softc.armoryexpansion.common.integration.aelib.plugins.constructsarmory.material;
+
+import org.softc.armoryexpansion.common.integration.aelib.config.MaterialConfigOptions;
+import org.softc.armoryexpansion.common.integration.aelib.plugins.tinkersconstruct.material.IToolMaterial;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.library.materials.ExtraMaterialStats;
+import slimeknights.tconstruct.library.materials.HandleMaterialStats;
+import slimeknights.tconstruct.library.materials.HeadMaterialStats;
+import slimeknights.tconstruct.library.materials.MaterialTypes;
+
+public class ArmorToolMaterial extends ArmorMaterial implements IToolMaterial {
+    protected HeadMaterialStats headMaterialStats;
+    protected HandleMaterialStats handleMaterialStats;
+    protected ExtraMaterialStats extraMaterialStats;
+
+    ArmorToolMaterial(String identifier, int color) {
+        super(identifier, color);
+    }
+
+    @Override
+    public HeadMaterialStats getHeadMaterialStats() {
+        return this.headMaterialStats;
+    }
+
+    @Override
+    public HandleMaterialStats getHandleMaterialStats() {
+        return this.handleMaterialStats;
+    }
+
+    @Override
+    public ExtraMaterialStats getExtraMaterialStats() {
+        return this.extraMaterialStats;
+    }
+
+    @Override
+    public boolean isToolMaterial() {
+        return null != this.headMaterialStats
+                || null != this.handleMaterialStats
+                || null != this.extraMaterialStats;
+    }
+
+    @Override
+    public boolean registerTinkersMaterialStats(MaterialConfigOptions properties){
+        slimeknights.tconstruct.library.materials.Material material = TinkerRegistry.getMaterial(this.getIdentifier());
+//        if (!"unknown".equals(material.getIdentifier())) {
+            if (properties.materialEnabled()) {
+                this.registerArmorStats(material, properties);
+                this.registerToolStats(material, properties);
+                return true;
+            }
+//        }
+        return false;
+    }
+
+    void registerToolStats(slimeknights.tconstruct.library.materials.Material material, MaterialConfigOptions properties){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896610)